src/index.ts

```diff
@@ -37,7 +37,11 @@ export interface WebResource {
 	size?: number;
 };
 `,
-	write: '',
+	write: `
+export interface WebResourceWrite extends Blob {
```

Comment:
I don't think this type is actually correct because you can send other things to the server right? eg based on https://github.com/balena-io-modules/sbvr-types/blob/v7.0.1/src/types/web-resource.ts#L94-L102 then either string or object is valid